tests/MSBuildDeviceIntegration/MSBuildDeviceIntegration.csproj

```diff
@@ -35,7 +35,7 @@
 
   <ItemGroup>
     <PackageReference Include="NodaTime" Version="2.4.5" />
-    <PackageReference Include="MSBuild.StructuredLogger" Version="2.1.500" />
+    <PackageReference Include="MSBuild.StructuredLogger" Version="2.1.758" />
```

Comment:
@grendello FYI, the `.binlog` format just changed in .NET 8. I remember you were using this library for something?

Reply:
Yeah, to find values of properties when preparing a project for native debugging/profiling. I'll update my reference, thanks :)